ui/hooks/gasFeeInput/useGasFeeInputs.test.js

```diff
@@ -350,4 +351,13 @@ describe('useGasFeeInputs', () => {
       expect(result.current.supportsEIP1559V2).toBe(false);
     });
   });
+
+  describe('editGasMode', () => {
+    it('should return editGasMode passed', () => {
```

Comment:
Shouldn't it say something like: 'should return editGasMode for Swaps'?

Reply:
This can actually be other gas modes also.